### PR TITLE
[WIP] #834 : Add upload date filter to media gallery grid

### PR DESCRIPTION
**Pull request by konarshankar07**
### Description (*)

This PR will add the upload date filter in media gallery grid
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#834: Add upload date filter to media gallery grid

### Manual testing scenarios (*)

1. Log in to Admin Panel
2. Select "Media Gallery Development" from the menu on the right
3. Expand filters section
### Expected Result
image

### CODE REVIEW

**Comment by sivaschenko** [comment]
Hi @konarshankar07 thanks for the Pull request! Does this filter require any updates on backend side?

**Comment by konarshankar07** [comment]
Hello @sivaschenko ,
Yes, we still need to implement the logic for the applied filter but as per ticket description I thought that I need to add the filter component in this PR and logic implementation in another PR. But I'll now work on filter implementation logic
Thanks

**Comment by sivaschenko** [comment]
@konarshankar07 the problem is that there is no way to test the filter unless it's implemented in the backend. So I think it should be added in this PR. Let me know if you need any help.

Same thing about https://github.com/magento/adobe-stock-integration/pull/843

**Comment by chalov-anton** [comment]
Please, add an ability to select Date Range.  Regarding the first item from the **Acceptance Criteria** of the Story https://github.com/magento/adobe-stock-integration/issues/756 , user should be able to _select a date range_  
Like, for example in the Filter from _Customers- All Customers_ 
dofb

**Comment by konarshankar07** [comment]
@chalov-anton
I have realized my mistake after reading AC of the mention story and will update the field in my next commit
Thanks

**Comment by sivaschenko** [comment]
Hi @konarshankar07 let put the development of pull request on hold until https://github.com/magento/adobe-stock-integration/pull/843 is merged

**Comment by sivaschenko** [comment]
Hi @konarshankar07 please follow these instructions to add value/filter to the media gallery grid https://github.com/magento/adobe-stock-integration/wiki/Adding-values-filters-to-media-gallery-grid

Upload date is reflected as `created_at` field of Media Gallery asset

File: MediaGalleryUi/view/adminhtml/web/css/source/_module.less [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+& when (@media-common = true) {
```
Comment (sivaschenko) [inline MediaGalleryUi/view/adminhtml/web/css/source/_module.less]: I don't think any special styles for media gallery are required

**Review by sivaschenko — CHANGES_REQUESTED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**